app/helpers/orders_helper.rb

```diff
@@ -45,6 +45,6 @@ def contact_info(contact_address)
   end
 
   def contact_name(contact)
-    [contact['FirstName'], contact['LastName']].reject(&:empty?).join(' ')
+    [contact['FirstName'], contact['LastName']].reject(&:empty?).join(' ') unless contact.nil?
```

Comment:
What is the default return value for a method? That would be returned here right? IF it's `nil` will that cause problems with whatever is calling this method? If it's just `<%= contact_name(@stuff) %>` then we'll be fine.